src/model/utils/insertcontent.js

```diff
@@ -327,6 +327,30 @@ class Insertion {
 
 		mergePosLeft.detach();
 		mergePosRight.detach();
+
+		// @param {Set} canMergeWith
+		// @param {module:engine/model/schema~Schema} schema
+		// @returns {Boolean}
+		function canMergeLeft( canMergeWith, schema ) {
```

Comment:
This function params are odd. It gets some here but also uses the `context` and `node`. I'd prefer to move these functions out of this method's body and have more clarity about what it uses.

Reply:
The way it's implemented currently you see function calls and miss the idea what these functions do.

Reply:
Done.